frontend/uddhar-frontend/src/coordinator/components/Content.jsx

```
@@ -56,17 +58,26 @@ const ContentSection = ({ currentEvent, onGoingDisasters }) => {
+      )}
       <section className="w-full h-auto mx-auto p-6 relative border-2 rounded-lg">
         <Toaster position="top-center" richColors />
-        <button className="absolute top-2 right-2 bg-red-500 hover:bg-red-700 text-white px-8 py-2 rounded cursor-pointer"
-          onClick={() => setShowAlert(true)}
-        >
-          End
-        </button>
-        <h1 className="text-3xl font-bold mb-2">
-          {title}
+        <div className="flex justify-end items-center gap-x-5 absolute top-4 right-4">
+          <button className="border-1 border-gray-300 p-1 rounded cursor-pointer hover:bg-gray-400/70" onClick={()=> setShowEditDialog(true)}><Pencil className="w-5 h-5"/></button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)